### PR TITLE
#449 Testimonials redesign

### LINKED ISSUE #449: Testimonials
# Description

Redesign the testimonials section.

### PR DESCRIPTION
**Pull request by Ke-bean**
# PR Description

This PR is for redesigning the Testimonials page

Screen Shot 2024-11-22 at 08 29 47

# Please check this Checklist before you submit your PR:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [x] My code generate no warnings
- [x] My test coverage meet the set test coverage threshold
- [x] There are no vulnerabilities
- [x] There are no conflicts with the base branch

### CODE REVIEW

File: .github/workflows/tests runner.yml [hunk 1]
```diff
@@ -23,7 +23,6 @@ jobs:
       - name: Check linting errors
         run: npm run lint
       - name: Run tests and generate coverage reports
```
Comment (aimedivin) [inline .github/workflows/tests runner.yml]: @Ke-bean , why is this run removed

**Review by aimedivin — CHANGES_REQUESTED** [review]
@Ke-bean, Remove that horizontal scroll bar and make sure translation for the testimonial section work as I was working before (it was fixed by @rwemakevin),
image

**Comment by Ke-bean** [comment]
> @Ke-bean, Remove that horizontal scroll bar and make sure translation for the testimonial section work as I was working before (it was fixed by @rwemakevin), image

> @Ke-bean , why is this run removed

Could you clarify which work you said you did before or @rwemakevin's work so that I can reference on what to change? @aimedivin

File: .github/workflows/tests runner.yml (hunk 1 above)
Comment (Ke-bean) [reply .github/workflows/tests runner.yml]: > @Ke-bean , why is this run removed

Was testing something and forget to bring back it, now it's fixed @aimedivin

**Review by elijahladdie — APPROVED** [review]

**Review by Calebgisa72 — APPROVED** [review]

**Review by aimedivin — APPROVED** [review]

**Pull request merged**